spec/models/items_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'pry-byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.